procdiscovery/pkg/process/process.go

```diff
@@ -8,9 +8,6 @@ import (
 	"io"
 	"os"
 	"strings"
-
-	"github.com/odigos-io/odigos/common/consts"
-	"github.com/odigos-io/odigos/common/envOverwrite"
```

Comment:
note: this was the main intent. to retire the `envOverwrite` package, we need to migrate all it's users to consume it via the distros.

here we are stop using it in one place, and consume further changes via distro getter which pulls it from the manifests